crates/api/src/v1/proof.rs

```diff
@@ -31,11 +31,11 @@ pub struct ConsistencyResponse {
 /// Represents an inclusion proof request.
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct InclusionRequest<'a> {
-    /// The checkpoint to check for inclusion.
-    pub checkpoint: Cow<'a, Checkpoint>,
-    /// The log leafs to check for inclusion.
-    pub leafs: Cow<'a, [LogLeaf]>,
+pub struct InclusionRequest {
+    /// The log length to check for inclusion.
+    pub log_length: RegistryIndex,
```

Comment:
It would be _nice_ to give this a separate type name e.g. `RegistryLen`, but not critical.

Reply:
Do you think better to have two type aliases? One for the log length and the other for indexes into the log? Or one with a better name?

Reply:
Two, since you can't generally use one where you need the other.